base-R.qmd

```diff
@@ -422,8 +422,10 @@ Another important member of the apply family is `tapply()` which computes a sing
 
 ```{r}
 diamonds |> 
-  group_by(cut) |> 
-  summarise(price = mean(price))
+  summarise(
+    .by = cut,
+    price = mean(price),
+  )
```

Comment:
> calculate the mean price by cut.

Just noting that this feels like a natural way to state this and you never say "group by" here either

Reply:
Agreed. My concern was that people won't even have the word "group" as part of their vocabulary. This may not be a huge concern, just something I realized getting lost in the articulation compared to the `group_by() |> summarize()` flow.